plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -304,6 +304,23 @@ public static void main(String[] args)
         }
     }
 
+    public static class DeltaLakeGlueQueryRunnerMain
+    {
+        public static void main(String[] args)
+                throws Exception
+        {
+            // Set AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables
```

Comment:
Nit: I see this comment repeated everywhere when we deal with testing on AWS infrastructure:
```
Requires AWS credentials, which can be provided any way supported by the DefaultProviderChain See https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html#credentials-default
```

`AWS_REGION` needs to be set as well